src/scripts/main.js

```diff
@@ -1,5 +1,38 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
```

Comment:
It's a good practice to check if the element was actually found. If the class name changes in the HTML, this line would throw an error. Consider adding a check to ensure that the spider element exists before proceeding.